src/dirs.rs

```diff
@@ -42,6 +31,25 @@ impl BatProjectDirs {
         })
     }
 
+    pub fn get_cache_dir() -> Option<PathBuf> {
+        // on all OS prefer BAT_CACHE_PATH if set
+        let cache_dir_op = env::var_os("BAT_CACHE_PATH").map(PathBuf::from);
+        if cache_dir_op.is_some() {
+            return cache_dir_op;
+        }
+
+        #[cfg(target_os = "macos")]
+        let cache_dir_op = env::var_os("XDG_CACHE_HOME")
+            .map(PathBuf::from)
+            .filter(|p| p.is_absolute())
+            .or_else(|| dirs_rs::home_dir().map(|d| d.join(".cache")));
+
+        #[cfg(not(target_os = "macos"))]
+        let cache_dir_op = dirs_rs::cache_dir();
+
+        Some(cache_dir_op.map(|d| d.join("bat")))?
```

Comment:
Couldn't this just be:
```rust
cache_dir_op.map(|d| d.join("bat"))
```

Using `Some(xx)?` is the same as `xx`, right?

Reply:
Good catch. That does appear to be the case. I was trying to maintain the `?` operator (since this part of the function was extracted from the `new() `function) but it appears that removing Some() and ? works the same because its already an Option.